### ISSUE TITLE
Tags switch from <p> to <div> when cleaning the content

### ISSUE THREAD

**Issue by panchazo** [psergus/ngWYSIWYG#25]
If you copy/paste couple lines of text (multiline text makes the issue more evident) from eg Notepad to the editor if it has a <p></p> tag inside it will use proper tags to generate lines, bold, etc (you can see this in the source view). If you clean all contents (to be sure go to source view and delete everything) it will generate <div> tags to create new blocks instead of using <p>.
By the way, the editor is awesome, excellent work!!

**Comment by psergus** [comment]
@panchazo, yeah you are right. It's due to the option styleWithCSS which is enabled by document.execCommand("styleWithCSS", 0, 0); you can comment this line and it will be generating tags in the old style. Here is some explanation: http://stackoverflow.com/questions/3088993/webkit-stylewithcss-contenteditable-not-working
I guess I will need to contemplate on adding this as an option to configure the editor.
